### PR TITLE
Seeding data

### PR DESCRIPTION
**Pull request by ValeriiaTen**
Screenshot from 2020-08-14 23-48-14
Screenshot from 2020-08-14 23-48-27

`const productSchema = mongoose.Schema({
  // TODO: your schema here!
  productId: { type: Number, unique: true },
  itemName: String,
  options: { size: [String], color: [String], quantity: [Number] },
  materials: String,
  itemDescription: String
});`

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -1,46 +1,23 @@
 const mongoose = require('mongoose');
-// Step 2 - if it is avaliable then use it, if not then use our default environment
-mongoose.connect(process.env.MONGODB_URI || 'mongodb://localhost/fetcher', { useMongoClient: true });
-const db = mongoose.connection;
+mongoose.connect('mongodb://localhost/productDescription', { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = global.Promise;
```
[the remark this answers is not in the document]
Comment (ValeriiaTen) [reply database/index.js:3]: Before I upgraded from 4 to 5 of the Mongoose, I thought I would use the Mongoose in a different position inside the codes, in which case it should have been treated as global mode, which is why I thought to set the mongoose as : mongoose.Promise = global.Promise;

File: database/seed.js [hunk 2]
```diff
@@ -0,0 +1,1882 @@
+const Description = require('./index.js');
```
[the remark this answers is not in the document]
Comment (ValeriiaTen) [reply database/seed.js:1]: Are you talking about this script "db:setup": "node database/seed.js"  ?

[the remark this answers is not in the document]
Comment (nickfreemandesign) [reply database/seed.js:1]: yes, my comment is inside the file you just mentioned. `seed.js` just contains a static collection of items with one insert into your database. instead, you can create a pipeline that generates _n_ number of items, and then inserts into your database.

[the remark this answers is not in the document]
Comment (ValeriiaTen) [reply database/seed.js:1]: thank you for your advice, I have already started working on this with faker, it is really convenient, and I almost manually created 100 elements

File: webpack.config.js [hunk 3]
```diff
@@ -1,4 +1,6 @@
 module.exports = {
+  /* eslint-disable no-path-concat */
+  /* eslint-disable prefer-template */
   entry: __dirname + '/client/src/index.jsx',
```
Comment (ttreit) [inline webpack.config.js:4]: Does this work ok without using Node's path module? I've only seen it done with path before.

File: database/seed.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+function productDetails() {
+  var data = [];
+  for (var i = 1; i < 101; i++) {
+    data.push({
+      productId: i,
+      itemName: faker.commerce.productName(),
+      options: {
+        valeriia: 'kool',
```
[the remark this answers is not in the document]
Comment (ValeriiaTen) [reply database/seed.js:18]: Oh, great, I added this line to check if my code works or not and forgot to delete it. Thank you for noticing.

File: .eslintrc.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    },
+    ecmaVersion: 11,
+  },
+  plugins: [
+    'react',
+  ],
+  rules: {
+    'no-console': 'off',
+  },
+};
```
[the remark this answers is not in the document]
Comment (ValeriiaTen) [reply .eslintrc.js:23]: eslint itself asks me to add a comma after

File: database/index.js [hunk 6]
```diff
@@ -1,46 +1,10 @@
 const mongoose = require('mongoose');
-// Step 2 - if it is avaliable then use it, if not then use our default environment
-mongoose.connect(process.env.MONGODB_URI || 'mongodb://localhost/fetcher', { useMongoClient: true });
-const db = mongoose.connection;
+mongoose.connect('mongodb://localhost/products', { useNewUrlParser: true, useUnifiedTopology: true, useCreateIndex: true });
```
[the remark this answers is not in the document]
Comment (ValeriiaTen) [reply database/index.js:2]: Yes, it worked like this, never thought about it, but I think it's a good idea to add a port number

File: .gitignore [hunk 7]
```diff
@@ -1 +1,4 @@
 node_modules
+.eslintignore
+controller
+client
```
[the remark this answers is not in the document]
Comment (ValeriiaTen) [reply .gitignore:4]: thanks for the idea!

**Pull request merged**